src/components/Home.jsx

```diff
@@ -1,3 +1,29 @@
+import { useEffect, useState } from "react";
+import { getArticles } from "../utils.js/api";
+import { ArticleCard } from "./ArticleCard";
+
 export const Home = () => {
-  return <h2>Homepage</h2>;
+  const [articles, setArticles] = useState([]);
+  const [totalCount, setTotalCount] = useState(0);
+  useEffect(() => {
+    getArticles().then((response) => {
+      setArticles(response.articles);
+      setTotalCount(response.totalCount);
+    });
+  }, []);
+  console.log(articles);
+  console.log(totalCount);
+  return (
+    <ul className="articles_list">
+      {articles.map((article) => {
+        return (
+          <div>
+            <li className="article_card" key={article.article_id}>
+              <ArticleCard article={{ article }} />
+            </li>
+          </div>
+        );
```

Comment:
Is your div being used? If not I'd remove it unless you plan to use it. 

If your li can live within the ArticleCard I would move it there too